frontend/src/modules/controllers/AlarmModal.tsx

```diff
@@ -70,12 +70,9 @@ export interface AlarmAdjustProps {
  * @prop {string} units Alarm paramater unit measurement to display
  * @prop {number} committedMin Lower Set Alarm Range Value
  * @prop {number} committedMax Upper Set Alarm Range Value
- * TODO: it was used in ValueModa in values page, can be removed
- * @prop {boolean} disableAlarmButton Toggle to show/hide alarm button
  * @prop {function} updateModalStatus Callback to send current modal open/close status
  * @prop {function} onModalClose Callback after Modal close
- * TODO: rename to 'getCommittedRange' as it just returns the committed values.
- * @prop {function} requestCommitRange Callback on updating the Alarm range values
+ * @prop {function} getCommittedRange Callback on updating the Alarm range values
```

Comment:
It looks like the function interface for `getCommittedRange` means that it doesn't actually return anything; the `useEffect` which uses it just runs `getCommittedRange` without assigning any return value from that function into anything. It seems like `getCommittedRange` is also a misleading name, or else maybe we can refactor the way we split up code into different functions so that `getCommittedRange` returns something which is then used by the `useEffect` which calls it? I'm not sure, we should discuss our options.

Reply:
Looks like `handleCommitRange` or `handleRangeCommit` or `handleCommittedRange` is more appropriate. This is used by MultiStepWizard, which provides `doSetAlarmValues` as the callback function for this.

Reply:
using `handleCommittedRange`